pkg/executables/helm.go

```diff
@@ -116,19 +118,26 @@ func (h *Helm) SaveChart(ctx context.Context, ociURI, version, folder string) er
 }
 
 func (h *Helm) InstallChartFromName(ctx context.Context, ociURI, kubeConfig, name, version string) error {
-	params := []string{"install", name, ociURI, "--version", version, "--kubeconfig", kubeConfig}
+	params := []string{"upgade", "--install", name, ociURI, "--version", version, "--kubeconfig", kubeConfig}
```

Comment:
Maybe a typo? upgrade instead?

Reply:
Indeed. Thank you for the keen eyes! Fixed.